tests/Guzzle5ExceptionTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Http\Adapter\Tests;
+
+use GuzzleHttp\Exception as GuzzleExceptions;
+use Http\Adapter\Guzzle5HttpAdapter;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class Guzzle5ExceptionTest extends \PHPUnit_Framework_TestCase
+{
+    public function testGetException()
+    {
+        $request = $this->getMock('Psr\Http\Message\RequestInterface');
```

Comment:
This package does not have a dependency on any implementation of Psr\Http\Message\RequestInterface. That is why I mock it. 

Should I require an implementation instead?